router/router_test.go

```diff
@@ -108,6 +110,105 @@ func TestHeadersFromConfiguration(t *testing.T) {
 	assert.Equal(t, "Nice", res.Header.Get("New-Cool-Header"))
 }
 
+func TestHeadersFromCORSConfig(t *testing.T) {
```

Comment:
Could you add a test with an invalid origin? It should return statuscode 403 I guess.